packages/vscode-dependencies-validation/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@sap-devx/vscode-dependencies-validation",
+  "displayName": "vscode-dependencies-validation",
+  "private": true,
+  "description": "",
+  "version": "1.3.0",
+  "engines": {
+    "vscode": "^1.62.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onCommand:vscode-dependencies-validation.displayIssues"
+  ],
+  "main": "./dist/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "vscode-dependencies-validation.displayIssues",
+        "title": "Display Issues"
+      }
+    ]
+  },
+  "scripts": {
```

Comment:
instead of generating from a template, I recommend you copy and modify 
from the existing VSCode ext in this mono-repo.

That way we will have a consistent project structure/scripts/flows

Reply:
Done